# HG changeset patch
# User Kim Alvefur <zash@zash.se>
# Date 1601928749 -7200
# Node ID e17b98feb0b7cad6539ee25c5cc7728911349a6d
# Parent  c3eefb517b7b934dbf4f78244c18f7e91b852846
util.dependencies: Check for bitop library same way as net.websocket.frames (fixes #1594)

diff -r c3eefb517b7b -r e17b98feb0b7 util/dependencies.lua
--- a/util/dependencies.lua
+++ b/util/dependencies.lua
@@ -90,7 +90,7 @@
 			}, "SSL/TLS support will not be available");
 	end
 
-	local bit = _G.bit32 or softreq"bit";
+	local bit = softreq"bit" or softreq"bit32";
 
 	if not bit then
 		missingdep("lua-bitops", {

